auth0/src/main/java/com/auth0/android/request/internal/UserProfileDeserializer.java

```diff
@@ -15,6 +16,13 @@
 import java.util.Map;
 
 class UserProfileDeserializer implements JsonDeserializer<UserProfile> {
+
+    private final Gson iso8601DateGson;
+
+    public UserProfileDeserializer() {
```

Comment:
@poovamraj - can you check that I'm initializing this correctly

Reply:
This is good enough @adamjmcgrath, we could inject it through constructor but at this point I don't see this affecting our testing and if refactor is required, we can do it without affecting the external API.